src/pages/cart/cart.view.ts

```diff
@@ -0,0 +1,54 @@
+import { Card } from '../../components/card/card.view';
+import { ProductInRow } from '../../components/productInRow/productInRow.view';
+import { Product } from '../../model/product.model';
+import './cart.style.css';
+
+export class Cart {
+    private readonly id: string;
+    productInRow: ProductInRow;
+    card: Card;
+    constructor(id: string) {
+        this.id = id;
+        this.productInRow = new ProductInRow('productsRows');
+        this.card = new Card('cartWrap');
+    }
+
+    render(productsCart: Product[]): void {
+        const body = <HTMLBodyElement>document.getElementById(this.id);
+        const cartWrap = <HTMLDivElement>document.createElement('div');
+        const productsInCart = <HTMLDivElement>document.createElement('div');
+        const productsTitle = <HTMLDivElement>document.createElement('div');
+        const productsRows = <HTMLDivElement>document.createElement('div');
+        const summary = <HTMLDivElement>document.createElement('div');
+        cartWrap.id = 'cartWrap';
+        productsInCart.className = 'productsInCart';
+        productsTitle.className = 'productsTitle';
+        productsRows.id = 'productsRows';
+        productsInCart.appendChild(productsTitle);
+        productsInCart.appendChild(productsRows);
+        cartWrap.appendChild(summary);
+        cartWrap.appendChild(productsInCart);
```

Comment:
Можно это все заменить на строку. так получится нагляднее 
```suggestion
               const cartWrap = <HTMLDivElement>document.createElement('div');
        cartWrap.id = 'cartWrap';
        cartWrap.innerHTML = `<div class="productsInCart">
          <div class="productsTitle">
            <span>Products In Cart</span>
            <div class="page"><div>LIMIT: 1</div>
                <div class="page_buttons">PAGE: <button><</button><span>1</span><button>></button></div>
            </div>
          </div>
          <div id="productsRows"></div>
        </div>`;

Reply:
done